docs/environment-variables.md

```diff
@@ -45,3 +67,19 @@ Note that these environment variables may be removed at any time.
 | `PNS_PRIVILEGED` | `bool` | Whether to always set privileged on for PNS when PNS executor is used. |
 | `REMOVE_LOCAL_ART_PATH` | `bool` | Whether to remove local artifacts. |
 | `RESOURCE_STATE_CHECK_INTERVAL` | `time.Duration` | The time interval between resource status checks against the specified success and failure conditions. |
+
+You can set the environment variables for executor by customizing executor container's environment variables in your
+controller's configmap like the following:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argo-controller-configmap
```

Comment:
Name is wrong. `workflow-controller-configmap`

Reply:
The name is flexible depending on what is specified in the controller's `--configmap` but I updated this to be consistent.